.idea/.name

```diff
@@ -0,0 +1 @@
+intl-tel-input
```

Comment:
I'm having trouble getting rid of this comment, I have added .idea to the .gitignore file, is there some other step to it?